lib/evercam_media/camera_status_automation.ex

```diff
@@ -0,0 +1,75 @@
+defmodule EvercamMedia.CameraStatusAutomation do
+  import EvercamMediaWeb.CameraController, only: [check_params: 1]
+  alias EvercamMedia.Util
+  alias EvercamMedia.XMLParser
+  alias EvercamMedia.HTTPClient
+  require Logger
+
+  def check_camera_status(camera) do
+    address = camera |> Map.get("address")
+    vh_port = camera |> Map.get("port")
+    nvr_port = camera |> Map.get("nvr_port")
+
+    vh_status = check_port_status(address, vh_port)
+    nvr_status = check_port_status(address, nvr_port)
+
+    do_action(camera, vh_status, nvr_status)
+  end
+
+  # Case 1 when VH:Closed, NVR:Open
+  defp do_action(camera, vh_status, nvr_status) when vh_status == false and nvr_status == true do
```

Comment:
Don't need to use `when` condition.
```suggestion
  defp do_action(camera, false, true) do
```